doc/intern.rst

```diff
@@ -184,7 +184,7 @@ How the RTL is compiled
 
 The ``system`` module contains the part of the RTL which needs support by
 compiler magic (and the stuff that needs to be in it because the spec
-says so). The C code generator generates the C code for it just like any other
+says so). The C code generator generates the C code for it's just like any other
```

Comment:
I'm not a native English speaker, but the original version (with `it`) sounds correct to me.

Reply:
I agree. "it's" is short for "it is", a phrase which is generally used to establish equivalency between two objects.

Reply:
Without the "'s" it sound incomplete — as in "for it [i.e. the code generator], just like any other module." — what about it?

I believe the sentence was intended to mean "for it IS just like any other module."

Reply:
I thought it is: "The C code generator generates the C code for it [i.e. the system module], just like any other module."

Reply:
In that case, the sentence might need rephrasing for it's ambiguous. I had also taken into consideration that meaning, but ultimately though it meant as I've corrected it, for it seems to make more sense from how the sentence is structured and the general speech expectations.

But you might be right and I misunderstood the sentence.

We'll need a confirmation from a Nim developer on this.

Reply:
@Araq, could you confirm the correct meaning of that sentence?

Reply:
The intended meaning is indeed:

"The C code generator generates the C code for it [i.e. the system module], just like any other module."

Reply:
Amended accordinly. The added comma dispels ambiguity of the sentence. Now it's:

> The C code generator generates the C code for it, just like any other module.

Thanks for catching this @narimiran, and thanks for the clarification @Araq.